test/unit/job_invocation_test.rb

```diff
@@ -61,6 +61,16 @@
         job_invocation.generate_description!
         job_invocation.description.must_equal "#{job_invocation.job_name} - %{missing_key}"
       end
+
+      it 'truncates generated description to 255 characters' do
+        limit = 255
+        JobInvocation.columns_hash['description'].expects(:limit).returns(limit)
+        job_invocation.expects(:save!)
+        job_invocation.description_format = '%{job_name}'
+        job_invocation.job_name = 'a' * 1000
+        job_invocation.generate_description!
+        job_invocation.description.must_equal 'a'*limit
```

Comment:
`'a' * limit` as few lines above, also maybe `job_invocation.description.lenght.must_equal limit` would be more descriptive